### ISSUE TITLE
Geometry.reference database does not consider scaling option

### ISSUE THREAD

**Issue by astumpf** [cyberbotics/urdf2webots#184]
Using the `Geometry.reference`, all shapes loaded from the same mesh are getting replaced even though they are mirrored (scaling factor -1.0 in one axis). This leads to wrong use as the newly introduced `ccw` flag which is part of the geometry and must be adapted based on the scaling factors.

This leads to wrong rendering as faces are pointing in the wrong direction:

### Correct faces:
```
children [
  Transform {
    scale 1.000000 -1.000000 1.000000
    children [
      DEF wheel_small_mesh_visual CadShape {
        url "<my_path>/wheel_small_mesh.obj"
        ccw FALSE
      }
    ]
  }
]
```

![Screenshot from 2022-09-30 13-42-51](https://user-images.githubusercontent.com/7815721/193266557-a9be0263-53f9-49bb-8a3f-7329eda31cfc.png)

### Wrong faces when mirrored (wrong ccw as forced copied from original mesh object):
```
children [
  USE wheel_small_mesh_visual
]
```

![Screenshot from 2022-09-30 13-41-59](https://user-images.githubusercontent.com/7815721/193266564-26b4aa45-0838-4abd-b87a-7374edbe2e94.png)

**Comment by astumpf** [comment]
Fix in #185

**Comment by omichel** [comment]
Fixed in https://github.com/cyberbotics/urdf2webots/pull/187.
